simplq/src/styles/buttons.module.scss

```diff
@@ -12,6 +12,7 @@
   text-align: center;
   transition: background 0.8s;
   background-position: center;
+  min-height: 40px;
```

Comment:
Hi @rasfa98 Thanks for the fix. We prefer to use `rem` throughout. Can you change to `2.5rem`?

Reply:
Will do! 🙂